src/llvm/lib_llvm.cr

```diff
@@ -19,6 +19,7 @@ end
 
 {% begin %}
   lib LibLLVM
+    IS_140 = {{LibLLVM::VERSION.starts_with?("14.0")}}
```

Comment:
I'd also add `IS_LT_140`.

Reply:
Apparently there's no need for that, so why bother?